### PR TITLE
Add dnnrly projects

### PR DESCRIPTION
**Pull request by dnnrly**

### CODE REVIEW

File: projects/dnnrly/abbreviate.mdx [hunk 1]
```diff
@@ -0,0 +1,43 @@
+---
+name: abbreviate
+repoUrl: https://github.com/dnnrly/abbreviate
+description: Supporting your devops by shortening your strings using common abbreviations and clever guesswork.
+languages: # A list of programming languages used in this project
+  - Go
+tags: # A list of keywords relevant to the contributors for this project
+  - First Timer Friendly
+  - Tools
+twitterUrl: pascaldoesgo
```
Comment (NJalal7) [inline projects/dnnrly/abbreviate.mdx:10]: Could please update the Twitter entries for your projects with the full URL? Thanks!

**Review by NJalal7 — CHANGES_REQUESTED** [review]
Hi Pascal, thank you for adding your projects to OSS Port. I've left a few minor comments, but the project pages are coming along really nicely—we're excited to help you find contributors soon!

**Review by shaneakr — APPROVED** [review]
Thanks so much for your contribution!

File: projects/dnnrly/hoofli.mdx [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+languages: # A list of programming languages used in this project
+  - Go
+tags: # A list of keywords relevant to the contributors for this project
+  - First Timer Friendly
+  - Tools
+twitterUrl: pascaldoesgo
+currentlySeeking: # (optional) These fields are searchable and very helpful for contributors to match well with your project
+  - Developers
+featuredMap: # (optional) A CodeSee Map to serve as a visual starting point for contributors
+  url: https://app.codesee.io/maps/d78e36e0-2173-11ec-9551-537c5ee7bea6
```
[the remark this answers is not in the document]
Comment (dnnrly) [reply projects/dnnrly/hoofli.mdx:14]: This has been rectified, please check again and make sure I've made it visible.

File: projects/dnnrly/abbreviate.mdx (hunk 1 above)
Comment (dnnrly) [inline projects/dnnrly/abbreviate.mdx:10]:
```suggestion
twitterUrl: https://twitter.com/pascaldoesgo
```

File: projects/dnnrly/hoofli.mdx [hunk 3]
```diff
@@ -0,0 +1,40 @@
+---
+name: hoofli
+repoUrl: https://github.com/dnnrly/hoofli
+description: Generate PlantUML diagrams from Chrome or Firefox network inspections.
+languages: # A list of programming languages used in this project
+  - Go
+tags: # A list of keywords relevant to the contributors for this project
+  - First Timer Friendly
+  - Tools
+twitterUrl: pascaldoesgo
```
Comment (ramonh) [inline projects/dnnrly/hoofli.mdx:10]:
```suggestion
twitterUrl: https://twitter.com/pascaldoesgo
```

File: projects/dnnrly/httpref.mdx [hunk 4]
```diff
@@ -0,0 +1,41 @@
+---
+name: httpref
+repoUrl: https://github.com/dnnrly/httpref
+description: Command line, offline, access to HTTP status code, common header, and port references.
+languages: # A list of programming languages used in this project
+  - Go
+tags: # A list of keywords relevant to the contributors for this project
+  - First Timer Friendly
+  - Tools
+twitterUrl: pascaldoesgo
```
Comment (ramonh) [inline projects/dnnrly/httpref.mdx:10]:
```suggestion
twitterUrl: https://twitter.com/pascaldoesgo
```

File: projects/dnnrly/hoofli.mdx (hunk 3 above)
Comment (dnnrly) [inline projects/dnnrly/hoofli.mdx]:
```suggestion
twitterUrl: https://twitter.com/pascaldoesgo
```

File: projects/dnnrly/puzzle-template.mdx [hunk 5]
```diff
@@ -0,0 +1,40 @@
+---
+name: puzzle-template
+repoUrl: https://github.com/dnnrly/puzzle-template
+description: A generic template for puzzle projects such as Project Euler or Advent of Code .
+languages: # A list of programming languages used in this project
+  - Go
+tags: # A list of keywords relevant to the contributors for this project
+  - First Timer Friendly
+  - Tools
+twitterUrl: pascaldoesgo
```
Comment (ramonh) [inline projects/dnnrly/puzzle-template.mdx:10]:
```suggestion
twitterUrl: https://twitter.com/pascaldoesgo
```

File: projects/dnnrly/httpref.mdx (hunk 4 above)
Comment (dnnrly) [inline projects/dnnrly/httpref.mdx]:
```suggestion
twitterUrl: https://twitter.com/pascaldoesgo
```

File: projects/dnnrly/puzzle-template.mdx (hunk 5 above)
Comment (dnnrly) [inline projects/dnnrly/puzzle-template.mdx]:
```suggestion
twitterUrl: https://twitter.com/pascaldoesgo
```

**Review by ramonh — APPROVED** [review]

**Comment by dnnrly** [comment]
Thank you all for your help!

❤️

**Pull request merged**